ts/src/bingx.ts

```diff
@@ -646,13 +654,13 @@ export default class bingx extends Exchange {
         const quote = this.safeCurrencyCode (quoteId);
         const currency = this.safeString (market, 'currency');
         const settle = this.safeCurrencyCode (currency);
-        let pricePrecision = this.safeInteger (market, 'pricePrecision');
+        let pricePrecision = this.safeNumber (market, 'tickSize');
```

Comment:
why did we change the order of the ifs here? we were reading pricePrecision first and then tickSize

Reply:
yes, i have changed it, because it shouldnt matter

Reply:
but i'll revert if you think we should retain same flow.